src/subscriptions/components/subsciptions/Subscriptions.tsx

```diff
@@ -132,7 +132,7 @@ function Subscriptions(): React.ReactElement {
   const handleCheckboxValues = (
     index: number,
     code?: string,
-    value?: boolean
+    value = false
```

Comment:
Can this still also have the explicit type? Something like this perhaps:

    value: boolean = false

Reply:
No, it raises an error. Setting a default value also types it. "value" can only be true / false after that.

Error: "Type boolean trivially inferred from a boolean literal, remove type annotation."

Reply:
Got it.